static/locales/en-US.yaml

```
@@ -1217,3 +1217,5 @@ KeyboardShortcutPrompt:
   Skip by Tenths: Skip through video by percentage (3 skips to 30% of duration)
   Home: Seek to the beginning of the video
   End: Seek to the end of the video
+  Skip to Next Video: Skip to the next video (when using a playlist OR when autoplay is enabled)
```

its okay to just keep it lowercase because its userfacing and not devs reading it

```suggestion
  Skip to Next Video: Skip to the next video (when using a playlist or when autoplay is enabled)
```

---

Done